docs/authentication/fb-google-oauth2.html

```diff
@@ -1,28 +1,360 @@
+      <pre><code class="language-swift">import Credentials
+import CredentialsFacebook
+import KituraSession</code></pre>
+  </div>
+  <div class="tabcontent googleTab">
+      <p>Add <a href="https://github.com/IBM-Swift/Kitura-Credentials" target="_blank">Kitura-Credentials</a>, <a href="https://github.com/IBM-Swift/Kitura-CredentialsGoogle" target="_blank">Kitura-CredentialsGoogle</a> and <a href="https://github.com/IBM-Swift/Kitura-Session" target="_blank">Kitura-Session</a> to the dependencies in the `Package.swift` file. </p>
+      <p>Inside the file which defines the routes, `OAuth2Routes.swift`, import Credentials, CredentialsGoogle and KituraSession:</p>
+      <pre><code class="language-swift">import Credentials
+import CredentialsGoogle
+import KituraSession</code></pre>
+  </div>
+  <div class="underline"></div>
+  <h2 class="heading-2"><span class="blue-text">Step 4:</span> Set up a session</h2>
+  <p class="block-text">Kitura-Credentials requires a session to be set up on the protected routes. This will use cookies to keep the user logged in after the initial authentication.</p>
+  <pre><code class="language-swift">let session = Session(secret: "AuthSecret", cookie: [CookieParameter.name("Kitura-Auth-cookie")])
+app.router.all("/oauth2", middleware: session)</code></pre>
+  <div class="underline"></div>
+  <h2 class="heading-2"><span class="blue-text">Step 5:</span> Configure your Credentials middleware</h2>
+  <p>We need to initialize an instance of our Credentials plugin using the id and secret from the OAuth2 application we created in step 1.</p>
+  <p>We add the following code to our `initializeOAuth2Routes` function, substituting in our own id and secret:</p>
+  <div class="tabcontent facebookTab">
+  <pre><code class="language-swift">let fbClientId = "&lt;Your Facebook App ID&gt;"
+let fbClientSecret = "&lt;Your Facebook App Secret&gt;"
+let fbCallbackUrl = "http://localhost:8080/oauth2/facebook"
+let fbCredentials = CredentialsFacebook(clientId: fbClientId, clientSecret: fbClientSecret, callbackUrl: fbCallbackUrl)</code></pre>
+  </div>
+  <div class="tabcontent googleTab">
+  <pre><code class="language-swift">let googleClientId = "&lt;Your Google Client Id&gt;"
+let googleClientSecret = "&lt;Your Google Client Secret&gt;"
+let googleCallbackUrl = "http://localhost:8080/oauth2/google"
+let googleCredentials = CredentialsGoogle(clientId: googleClientId, clientSecret: googleClientSecret, callbackUrl: googleCallbackUrl)</code></pre>
+  </div>
+  <div class="underline"></div>
+  <h2 class="heading-2"><span class="blue-text">Step 6:</span> Register your Credentials plugin</h2>
+  <p>We need to register our plugin credentials to a `Credentials` middleware.</p>
+  <div class="tabcontent facebookTab">
+  <pre><code class="language-swift">let fbCredMiddleware = Credentials(options: ["successRedirect": "/oauth2/loggedin"])
```

Comment:
The redirect needs to change as we don't have an "/oauth2/loggedin" route! (this will also need to change in the complete code sample below and presumably in the Facebook plugin as well).